framework/include/utils/SolutionInvalidity.h

```diff
@@ -122,6 +144,11 @@ class SolutionInvalidity : protected ConsoleStreamInterface, public libMesh::Par
   /// Build a VariadicTable for solution invalidity
   FullTable summaryTable() const;
 
+  typedef VariadicTable<std::string, std::string, unsigned long int, unsigned long int> TimeTable;
+
+  /// Build a VariadicTable for solution invalidity history
+  TimeTable transientTable(unsigned int & time_interval) const;
```

Comment:
```suggestion
  TimeTable transientTable(const unsigned int time_interval) const;
```
no need to pass trivially copyable objects by reference